discord/colour.py

```diff
@@ -232,7 +245,26 @@ def blurple(cls):
 
     @classmethod
     def greyple(cls):
-        """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``."""
+        """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``.
+        .. versionchanged:: 1.5
+            Added grayple alias.
+        """
         return cls(0x99aab5)
 
+    grayple = greyple
```

Comment:
The `greyple` name is the branding one: https://discord.com/branding

Reply:
Ah, was not aware of that.